File: Java/1957. Delete Characters to Make Fancy String.java
class Solution {
    public String makeFancyString(String s) {
        StringBuilder sb = new StringBuilder();
        int size = 0;
        for (char c: s.toCharArray()) {
            if (size >= 2 && c == sb.charAt(size - 1) && c == sb.charAt(size - 2)) {
                continue;
            }
            sb.append(c);
            size++;
        }
        return sb.toString();
    }
}

File: Python3/1957. Delete Characters to Make Fancy String.py
class Solution:
    def makeFancyString(self, s: str) -> str:
        ans = []
        for c in s:
            if len(ans) >= 2:
                if c == ans[-1] == ans[-2]:
                    continue
            ans.append(c)

        return ''.join(ans)